### PR TITLE
Add .nvmrc file

### PR DESCRIPTION
**Pull request by psmarcin**
I've noticed that you use `nvm` but `.nvmrc` file is missing. It's good idea to easily keep the same node version for all developer. 

I don't want to use any alias for versioning. I used current version and just omit patches. 

More details: https://github.com/creationix/nvm#nvmrc

### CODE REVIEW

**Comment by znarf** [comment]
Thank you, definitely up for the .nvmrc.

The platform we're using for production (now) was only supporting 9.x.x so far. But good news, I just checked and 10.x.x is supported now.

Do you mind if we keep the version in `.travis.yml`? To explicitely keep everything in sync.

**Comment by psmarcin** [comment]
Yup, no problem. But I'm not sure if it's a good idea to use `v10` in development environment and run tests in `v9`. If you prefer to stick with `v9` for Travis I think it's fine but let's set the same version in `.nvmrc`. 

To sum up: should I set `v9` for both Travis and nvm or just Travis?

**Comment by znarf** [comment]
We can have version 10 everywhere, incuding travis.

It's just that in the current state of the PR, there is no version defined in travis (because you removed it).

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The official BackYourStack service is available from https://backyourstack.com/
 
 ## Development
 
-Make sure you have Node.js version >= 9. We recommend using [nvm](https://github.com/creationix/nvm).
+Make sure you have Node.js version >= 9. We recommend using [nvm](https://github.com/creationix/nvm): `nvm use`.
```
Comment (znarf) [inline README.md]: Maybe, we can also advertise version >= 10 here.

**Comment by psmarcin** [comment]
I removed node version from travis file because of this: 

> The .nvmrc file is only read when node_js key in your .travis.yml files does not specify a nodejs version.

More: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions-using-nvmrc

**Comment by znarf** [comment]
Cool, thank you for pointing on the travis documentation, I was not aware of that behavior.

Looks good!

**Pull request merged**